### DISCUSSION TITLE
Failed to initialize genesis miner

### DISCUSSION THREAD

**Discussion by HtopH** [filecoin-project/lotus#10986] [Build and Sync]
when i use the command
./lotus-miner init --genesis-miner --actor=t01000 --sector-size=2KiB --pre-sealed-sectors=~/.genesis-sectors --pre-sealed-metadata=~/.genesis- sectors/pre-seal-t01000.json --nosync
  When initializing the miner, an error appears in the log:
ERROR miner miner/miner.go:473 completed mineOne {"tookMilliseconds": 28, "forRound": 1, "baseEpoch": 0, "baseDeltaSeconds": 1775, "nullRounds": 0, "lateStart": true, "beaconEpoch ": 3040259, "lookbackEpochs": 900, "networkPowerAtLookback": "40960", "minerPowerAtLookback": "40960", "isEligible": true, "isWinner": true, "error": "failed to compute winning post proof: faulty sectors [SectorId(1)]", "errorVerbose": "failed to compute winning post proof:\n github.com/filecoin-project/lotus/miner.(*Miner).mineOne\n /home/admin/Data /softWare/lotus-local-net/miner/miner.go:543\n - faulty sectors [SectorId(1)]"}

Then I found that
 ./lotus-miner sectors list returns empty, the miner is not located in the sealed sector, 
but I have run
./lotus-seed pre-seal --sector-size 2KiB --num-sectors 2
Come to pre-seal 2 sectors

**Comment by rjan90** [comment]
Hey @HtopH! 

I suspect that you are seeing the above errors because you have potentially installed lotus/lotus-miner installs on different paths on your machine? With regards to the genesis-sectors/pre-seals they also need to be cleaned up on every reset of a local network. I would suggest to do a round of cleanup of the local dev-net repos:
- `rm -rf ~/.lotus-local-net`
- `rm -rf ~/.lotus-miner-local-net`
-  `rm -rf ~/.genesis-sectors`

And then start the local devnet tutorial from start again: https://lotus.filecoin.io/lotus/developers/local-network/#set-up-a-lotus-node

**Reply by HtopH** [reply]
Yes, I have downloaded several versions of Lotus, but the compiled executable files are all in different directories, and I have not put these executable files in the environment variables. Does this also affect it?

~/.lotus-local-net
~/.lotus-miner-local-net
~/.genesis-sectors
I will delete these directories every time I try again, but I have tried several times and this error is reported. I don’t know where the problem is.

**Reply by TippyFlitsUK** [reply]
Did this resolve your issue @HtopH?

**Comment by CodeIsLaw0108** [comment]
lotus-seed pre-seal --sector-size 2KiB --num-sectors 2 
would get file "pre-seal-f01000.json" instead of "pre-seal-t01000.json"

![image](https://github.com/filecoin-project/lotus/assets/81554458/1cbfb000-c10b-45fe-ba0c-12ddcb9eac72)

Suggest change it on lotus docs
